docs/topic-guides/blendable-web-apps/common-look-and-feel.md

```diff
@@ -0,0 +1,84 @@
+# Common look and feel of apps
+
+## Introduction
+
+Good user experience is significant for a successful solution of apps. An author of a single app aims to balance a good default look that matches other apps and to have distinctive features that are both easy to use and pleasing to the eye of the end user.
+
+Starcounter offers a layered approach that allows app authors to adapt to a common look and feel for the typical parts of the UI and to provide their own components for the unique parts. Solution owners can theme or replace the look and feel without changing of the app source code.
+
+## Mental model of client-side rendering by the Starcounter host
+
+Starcounter host follows a standards-based layered approach that separates the responsibilities of the app author, the design system, and the solution owner. 
+
+The full design consists of:
+
+1. Content defined by native HTML
+2. Common UI patterns and components of Uniform design system
+3. Theme configuration through custom CSS properties
+4. Extend or replace the theme with HTML, CSS, and JavaScript
+
+### Content defined by native HTML
+
+As an app author, you provide the views for your apps with a clear separation of the content and the presentation.
+
+In the content part, you provide the native semantic HTML elements that can be rendered on the screen directly or with their appearance changed in the presentation layer.
+
+### Common UI patterns and components of Uniform design system
+
+As the basic way for providing a common look and feel in the presentation layer, Starcounter provides Uniform design system. It consists of custom CSS properties, CSS reset and a library of patterns, and components.
+
+Uniform follows the inversion of control principle, which makes it a link between app authors and solution owners.
+
+Unlike many design systems, Uniform is open-ended and themeable. It can be configured, extended or replaced in parts to achieve anything that is possible with HTML, CSS, and JavaScript.
+
+The custom CSS properties are the basic set of primitives. They define the fonts, sizes, colors, margins and other values used throughout the reset, patterns and components.
+
+The CSS reset overwrites the default user agent stylesheet. It provides the look of the native HTML5 elements such as the headings (H1-H6), links, inputs, buttons, and tables. It is implicitly loaded by the Starcounter app shell, so there is no need to load it explicitly in the views. 
+
+The pattern library, [Uniform.css](https://starcounter.github.io/uniform.css/), is a set of CSS classes that implement visual hierarchy and give specific roles to native elements. It provides sections, cards, titles, column layouts, alerts,  buttons, etc.
+
+The component library, [Uniform components](https://starcounter.github.io/uniform.css/components/), is a set of custom elements that implement interactive UI controls. For the beginning, we provide a form item, a date picker, a data table and a pagination control. 
```

Comment:
Can you try to rephrase  "For the beginning, we provide..."? 

Maybe something like: "The first version includes a form item, data picker, database table, and pagination control."